content/graph/MaximumClique.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * Author: chilli, Haobin Ni
+ * Date: 2019-05-10
+ * License: CC0
```

Comment:
is CC0 true? if not, just remove this line

Reply:
Yeah it's also from the SJTU notebook.